### PR TITLE
fix(actions): Misspelled action levels no longer treated as logged_in.

### LINKED ISSUE #8337: Actions should validate access level on registration
E.g. the level "Admin" is silently treated as "logged_in".

### PR DESCRIPTION
**Pull request by mrclay**
Instead we now log an error and set the level to “admin”.

Fixes #8337

(Note this doesn't add constants for levels, as I'd prefer not to add global constants, nor ask devs to use constants on a private class we might refactor.)

### CODE REVIEW

**Comment by mrclay** [comment]
Or do we think an invalid level should throw an exception or refuse to register the action at all?

**Comment by hypeJunction** [comment]
IMHO, an exception is better than a surprise behavior

**Comment by mrclay** [comment]
I'd agree if this were a new function, but it seems unreasonable to WSOD over this.

File: engine/classes/Elgg/ActionsService.php [hunk 1]
```diff
@@ -32,7 +42,7 @@ class ActionsService {
 	 */
 	public function execute($action, $forwarder = "") {
 		$action = rtrim($action, '/');
-		$this->currentAction = $action;
+		$this->current_action = $action;
```
Comment (ewinslow) [inline engine/classes/Elgg/ActionsService.php]: Why all the underscores? When did this coding standard change?

Comment (mrclay) [inline engine/classes/Elgg/ActionsService.php]: http://learn.elgg.org/en/1.11/contribute/code.html#naming

Comment (ewinslow) [inline engine/classes/Elgg/ActionsService.php]: Doh. How long has that been that way? Clearly haven't been following it...
(and don't really want to, honestly!)

**Comment by mrclay** [comment]
Got rid of the renaming

**Comment by juho-jaakkola** [comment]
I have mild preference for an exception. But not blocking.

**Pull request merged**